package.json

```diff
@@ -40,18 +40,22 @@
     ]
   },
   "scripts": {
-    "karma": "karma start vision/explore-api/karma.conf.js",
+    "karma_speech": "karma start speech/explore-api/karma.conf.js",
+    "karma_vision": "karma start vision/explore-api/karma.conf.js",
     "lint": "semistandard \"**/*.js\"",
-    "test": "npm run lint && npm run karma"
+    "test": "npm run lint && npm run karma_speech && npm run karma_vision"
   },
   "devDependencies": {
     "chai": "^3.5.0",
+    "jasmine-core": "^2.5.2",
     "karma": "^1.1.1",
     "karma-chai": "^0.1.0",
     "karma-chrome-launcher": "^1.0.1",
     "karma-coverage": "^1.1.0",
     "karma-detect-browsers": "^2.1.0",
     "karma-firefox-launcher": "^1.0.0",
+    "karma-jasmine": "^1.1.0",
+    "karma-jasmine-ajax": "^0.1.13",
```

Comment:
I recommend using SinonJS instead of Jasmine to get the ajax mocking. Remove the three new dependencies and add `sinon` and `karma-sinon`. Here are the usage docs: http://sinonjs.org/docs/#server